git_theta/lsh/euclidean_lsh.py

```diff
@@ -68,8 +68,9 @@ def nb_hash(
 
 
 def get_lsh():
-    # TODO we need a better way of keeping track of configuration at the repository level
-    # For LSH configuration, once it is set for a repository, changing it should be handled with care
+    repo = git_utils.get_git_repo()
+    thetaconfig = config.ThetaConfigFile(repo)
```

Comment:
I think we want to make sure that we decouple a lot of these functions from needing there to be an actual config, otherwise they get really hard to reuse or use in tests.

We probably want things like the signature size, atol, etc to be parameters where None values result in looking it up in the config?